hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/IsAuthorizedSuite.java

```diff
@@ -0,0 +1,572 @@
+                                .key(THRESHOLD_KEY)
+                                .exposingCreatedIdTo(id -> accountNum.set(idAsHeadlongAddress(id))),
+                        cryptoCreate(ANOTHER_ACCOUNT)
+                                .balance(ONE_MILLION_HBARS)
+                                .key(KEY_LIST)
+                                .exposingCreatedIdTo(id -> accountAnotherNum.set(idAsHeadlongAddress(id))),
+                        uploadInitCode(HRC632_CONTRACT),
+                        contractCreate(HRC632_CONTRACT))
+                .when(withOpContext((spec, opLog) -> {
+                    final var messageHash = new Digest().digest("submit".getBytes());
+                    final var edKey = spec.registry().getKey(ED25519_KEY);
+                    final var privateKey = spec.keys()
+                            .getEd25519PrivateKey(com.swirlds.common.utility.CommonUtils.hex(edKey.toByteArray())
+                                    .substring(4));
+                    final var signedBytes = SignatureGenerator.signBytes(messageHash, privateKey);
+
+                    final var callWithThreshold = contractCall(
+                                    HRC632_CONTRACT, "isAuthorizedRawCall", accountNum.get(), messageHash, signedBytes)
+                            .via("authorizeCallWithThreshold")
+                            .gas(2_000_000L)
+                            .hasKnownStatus(CONTRACT_REVERT_EXECUTED);
+
+                    final var callWithKeyList = contractCall(
+                                    HRC632_CONTRACT,
+                                    "isAuthorizedRawCall",
+                                    accountAnotherNum.get(),
+                                    messageHash,
+                                    signedBytes)
+                            .via("authorizeCallWithKeyList")
+                            .gas(2_000_000L)
+                            .hasKnownStatus(CONTRACT_REVERT_EXECUTED);
+                    allRunFor(spec, callWithThreshold, callWithKeyList);
+                }))
+                .then(
+                        childRecordsCheck(
+                                "authorizeCallWithThreshold",
+                                CONTRACT_REVERT_EXECUTED,
+                                recordWith().status(FAIL_INVALID)),
+                        childRecordsCheck(
+                                "authorizeCallWithKeyList",
+                                CONTRACT_REVERT_EXECUTED,
+                                recordWith().status(FAIL_INVALID)));
+    }
+
+    @HapiTest
+    final Stream<DynamicTest> isAuthorizedRawAccountAndSignatureKeysCombinations() {
+        final AtomicReference<Address> accountNum = new AtomicReference<>();
+
+        return propertyPreservingHapiSpec("isAuthorizedRawAccountAndSignatureKeysCombinations")
+                .preserving(CONTRACTS_SYSTEM_CONTRACT_ACCOUNT_SERVICE_IS_AUTHORIZED_ENABLED)
```

Comment:
You could use the `@BeforeAll`/`@AfterAll` pattern like [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/SelfDestructSuite.java#L76) to only override this property once for the entire test class and remove override at the end.

But that can be a separate PR, the tests all make sense.